src/m365/aad/commands/app/app-role-delete.ts

```diff
@@ -0,0 +1,265 @@
+import { Cli, Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+import {Application, AppRole} from "@microsoft/microsoft-graph-types";
+import Utils from '../../../../Utils';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  appId?: string;
+  appObjectId?: string;
+  appName?: string;
+  claim?: string;
+  name?: string;
+  id?:string;
```

Comment:
Let's format the code (Shift+Alt+F on Windows or Shift+Option+F on mac in VSCode) to ensure consistent spacing throughout.

Reply:
Fixed